Improve sales val dbt docs (#607)
ccao-data/data-architecture: dbt/models/shared_columns.md

{% docs shared_column_is_ahsap %}
Affordable Housing Special Assessment Program indicator. For more information on

> AHSAP, see: <https://www.cookcountyassessor.com/affordable-housing>

Author: Maybe a pre-commit thing is doing this. I can remove the arrows and `--no-verify`?

Reviewer: I think it's fine to keep the arrows!

This variable combines `sv_is_heuristic_outlier`
with `sv_is_ptax_outlier` (using OR logic).

A null value represents an observation that, due to
filters on type of sale or time frame of sale, is
excluded completely from the sales-val model pipeline

> and therefor does recieve a boolean value.

Reviewer: **[Nitpick, non-blocking]** Two small typos here:

```suggestion
and therefore does receive a boolean value.
```

> NOTE: Outlier flags only exist for sales _after_ 2014.

Reviewer: **[Suggestion, non-blocking]** Maybe we could make the connection here explicit:

```suggestion
NOTE: Outlier flags only exist for sales _after_ 2014.
Sales before 2014 will have a null value for this column.
```

Also, is the date range inclusive of 2014? If not, the above text should read "Sales _including and_ before 2014".

repository for a list of possible flags.
{% enddocs %}

## sv_outlier_reason1

{% docs shared_column_sv_outlier_reason1 %}
One of three possible reasons that a sale is
flagged as on outlier. The priority for
sv_outlier_reason$n column filling is

> ptax outlier > price outlier > characteric outlier.

Reviewer: **[Nitpick, non-blocking]** Small typo here, `characteric` should be `characteristic`. No suggestion for this change, since I include this fix in my bigger-picture suggestion below.

> ## sv_outlier_reason1
> 
> {% docs shared_column_sv_outlier_reason1 %}
> One of three possible reasons that a sale is
> flagged as on outlier. The priority for
> sv_outlier_reason$n column filling is
> ptax outlier > price outlier > characteric outlier.
> 
> See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
> repository for a list of possible flags.
> {% enddocs %}
> 
> ## sv_outlier_reason2
> 
> {% docs shared_column_sv_outlier_reason2 %}
> One of three possible reasons that a sale is
> flagged as on outlier. The priority for
> sv_outlier_reason$n column filling is
> ptax outlier > price outlier > characteric outlier.
> 
> See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
> repository for a list of possible flags.
> {% enddocs %}
> 
> ## sv_outlier_reason3
> 
> {% docs shared_column_sv_outlier_reason3 %}
> One of three possible reasons that a sale is
> flagged as on outlier. The priority for
> sv_outlier_reason$n column filling is
> ptax outlier > price outlier > characteric outlier.
> 
> See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
> repository for a list of possible flags.
> {% enddocs %}

Reviewer: **[Nitpick, blocking]** Per my comment above, we can collapse these into one description given that they share the exact same text:

```suggestion
## sv_outlier_reason

{% docs shared_column_sv_outlier_reason %}
One of three possible reasons that a sale is
flagged as on outlier. The priority for
sv_outlier_reason$n column filling is
ptax outlier > price outlier > characteristic outlier.

See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
repository for a list of possible flags.
{% enddocs %}
```
